eth2/fork_choice/src/optimised_lmd_ghost.rs

```diff
@@ -30,10 +30,8 @@ use fast_math::log2_raw;
 use std::collections::HashMap;
 use std::sync::Arc;
 use types::{
-    readers::BeaconBlockReader,
-    slot_epoch_height::{Height, Slot},
-    validator_registry::get_active_validator_indices,
-    BeaconBlock, Hash256,
+    readers::BeaconBlockReader, slot_epoch::Slot, slot_height::SlotHeight,
```

Comment:
IMO we should just import `Slot` and `SlotHeight` directly from the root of `types`.

Reply:
Agreed